components/shared/right-sidebar/SidebarRelatedPosts.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Link from 'next/link';
+
+import PlusIcon from '../Icons/PlusIcon';
+
+
+const SidebarRelatedPosts = () => {
+  // Hardcoded placeholder for actual realated posts
+  const relatedPosts = ['Bottom Icons', 'Popup', 'Slider Sheet', 'Tab bar']
+
+  return (
+    <div className='flex flex-col gap-4'>
+        <h3 className='paragraph-2-bold border-white-500 border-b pb-4'>Related Posts</h3>
+
+        {relatedPosts.map((tag) => {
+            return (
+              <div key={tag} className=''>
+                <p className='text-white-300 paragraph-2-regular'>
+                  {tag}
+                </p>
+              </div>
+            )
+          })}
+
+      <Link href='/posts/create-post'>
+        <button className="paragraph-3-medium bg-black-600 flex w-full flex-row items-center justify-center rounded py-3 font-bold text-white">
+          <PlusIcon className='fill-primary-500'/>
+          <span className='ml-2'>Add related post</span>
+        </button>
```

Comment:
I would use a gap here- but this is much close to preference than the last component:

In case you have trouble finding the info on the figma:
https://github.com/DDVVPP/GitNote/assets/4108484/aeef2ca3-0f61-4a49-8b1e-04a964fb9149

We don't need flex-row, as that's the default flex.
It should be `py-2` not `py-3`
The class you have for the white color is `text-white-100` not `text-white`
![image](https://github.com/DDVVPP/GitNote/assets/4108484/5f2c5708-8ce4-4ec6-b514-e3266947d8ca)


### Important changes:
```suggestion
        <button className='paragraph-3-medium flex w-full items-center justify-center rounded bg-black-600 py-2 font-bold text-white-100'>
          <PlusIcon className='fill-primary-500' />
          <span className='ml-2'>Add related post</span>
        </button>
```

### Suggested Gap Version
```js
        <button className="paragraph-3-medium flex w-full items-center justify-center gap-2 rounded bg-black-600 py-2 font-bold text-white-100">
          <PlusIcon className="fill-primary-500" />
          Add related post
        </button>
```